docs/reference/controls/slider.md

```diff
@@ -38,6 +39,45 @@ The slider looks like this on Windows:
 
 <img src={SliderScreenshot} alt="" />
 
+## Binding to TextBox
+In this example, the slider value is bound to the text box above using binding to the control.
+
+```xml
+<StackPanel>
+  <TextBlock Text="Damage: " />
+  <TextBox Text="{Binding Damage}" />
+  <TextBlock Text="MaxDamage: " />
+  <TextBox Text="{Binding MaxDamage}" />
+  <Slider Maximum="{Binding MaxDamage}" Value="{Binding Damage}" />
+  <Button Command="{Binding UnlimitedDamage}" Content="∞" />
+</StackPanel>
+```
+```cs
+using System.Windows.Input;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace AvaloniaControls.ViewModels;
+
+public class MainViewModel : ViewModelBase
+{
+    [Reactive]
```

Comment:
ReactiveUI.Fody is a bit too specific, and we haven't really mentioned it before in the documentation. We should either omit VM impl details or add at least some comments about Fody.

Reply:
I would like to document a sliderbar example. 
I'd like to avoid relying on a particular framework, but then I won't be able to provide useful examples of how to do the binding.
Therefore, I had to explain it in a little more detail.